File: ui数学/矩形居中.c

struct position 
{
    float x;
    float y;
};
struct position juzhong(ParentWdith,ParentHeight,rectWdith,rectHeight)
{
    struct position pos;
    pos.x = (ParentWdith - rectWdith)/2;
    pos.y = (ParentHeight - rectHeight)/2;
    return pos;
}

File: ui数学/矩形居中.java
class 矩形居中{
    class Position{
        public float x;
        public float y;
    }
    public static void juzhong(ParentWidth,ParentHeight,rectWidth,rectHeight)
    {
        float x = (ParentWidth - rectWidth)/2;
        float y = (ParentHeight - rectWidth)/2;
        return new Position(x,y);
    }
}